packages/components/src/internal/components/EditInlineField.tsx

```diff
@@ -154,8 +160,11 @@ export const EditInlineField: FC<Props> = memo(props => {
                         name={name}
                         ref={inputRef}
                         type="text"
+                        size={value?.length || 20}
```

Comment:
if the initial value is just a couple of characters, does this small size value look odd? should we have a min size?

Reply:
After manual testing, I would probably learn towards the min of 20. But fine either way.

Reply:
Adjusted to minimum size of 20.